src/components/PasswordReset/actions.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import { RESET_PASSWORD, RESET_PASSWORD_SUCCESS, RESET_PASSWORD_FAILURE } from './constants';
+
+export const resetPassword = () => ({
+  type: RESET_PASSWORD,
+});
+
+export const resetPasswordSuccess = payload => ({
+  type: RESET_PASSWORD_SUCCESS,
+  payload,
+});
+
+export const resetPasswordFailure = errors => ({
+  type: RESET_PASSWORD_FAILURE,
+  errors,
+});
+
+const sendEmailToUser = data => (dispatch) => {
+  dispatch(resetPassword());
+  axios
+    // .post('https://ah-codeofduty-staging.herokuapp.com/api/users/forgot-password/', data)
+    .post('http://127.0.0.1:8000/api/users/forgot-password/', data)
```

Comment:
this url hits the localhost instead of heroku

Reply:
fixed